### DISCUSSION TITLE
can i use a const in schema.prisma?

### DISCUSSION THREAD

**Discussion by kin-for-test** [prisma/prisma#19263] [Q&A] [answered]
just like:

```
datasource db {
  provider = "mysql"
  url      = env("MYSQL_URL")
}

generator client {
  provider        = "prisma-client-js"
}

const idLength=21

model A{
  Id String          @db.Char(idLength)
}

model B{
  Id String          @db.Char(idLength)
}
```

**Comment by ludralph** [comment, marked as answer]
Hi @kin-for-test 👋 

It is not possible to use a `const` in a Prisma schema file. The schema file is a special file that makes use of the Prisma Schema Language (PSL). It mainly used as the configuration file for your Prisma setup. This [section](https://www.prisma.io/docs/concepts/components/prisma-schema) of the documentation shows you what is possible within the `schema.prisma` file.

**Reply by ludralph** [reply]
Hey @kin-for-test 

do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!
